backend/tests/CandidateDeciderDao.test.ts

```diff
@@ -37,3 +40,26 @@ test('Delete instance', () => {
     expect(allInstances.every((instance) => instance !== mockCDI));
   });
 });
+
+test('Update instance with transaction', async () => {
+  const promises: Promise<CandidateDeciderInstance>[] = [];
+
+  // Execute multiple concurrent update operations
+  const mockIDs = [1, 2, 3, 4, 5];
+  for (const mockID of mockIDs) {
+    promises.push(
+      candidateDeciderDao.updateInstanceWithTransaction(
+        mockCDI,
+        mockUser,
+        mockID,
+        mockRating,
+        mockComment
+      )
+    );
+  }
+
+  const results = await Promise.all(promises);
+
+  // Verify proper number of candidates are rated
+  expect(results[4].candidates.length === 5);
```

Comment:
maybe also verify that for two updates to the same ID, that the latest rating and comment will persist?